.github/workflows/migration-tests.yml

```diff
@@ -265,6 +265,7 @@ jobs:
           terraform_wrapper: false    
       - name: Migration Tests
         env:
+          MONGODB_ATLAS_ENABLE_PREVIEW: ${{ vars.MONGODB_ATLAS_ENABLE_PREVIEW }}
           MONGODB_ATLAS_ENABLE_BETA: ${{ vars.MONGODB_ATLAS_ENABLE_BETA }}
```

Comment:
Why are we retaining `MONGODB_ATLAS_ENABLE_BETA ` here but not in the other workflows?

Reply:
Here we are using an earlier version of the atlas terraform provider to check the migration tests. That earlier version will [fail](https://github.com/mongodb/terraform-provider-mongodbatlas/actions/runs/8231528028/job/22507253291)  if the variable is missing. 
For the other acceptance tests they all use the current version of the provider; therefore, they can use the new env-variable name

Reply:
that's ok

another alternative is to use SkipIfVersionBelow and/or IsProviderVersionAtLeast so we don't run those mig tests in prev. version to this one.

Reply:
Added a comment in a2bc4ed

Reply:
@maastha ok to resolve?